lib/user_interface.rb

```diff
@@ -84,16 +76,18 @@ def play
   end
 
   def new_score
-    puts message(:new_score).red.bold \
+    puts ' New Score !!'.red.bold \
       if @board.add(@name, @player.tries, @player.time)
+    PressAnyKey.new
+    UserInterface.new
   end
 
   def check_winner
```

Comment:
Is the name `check_winner` really checking for a winner?  Or is it the `report_win` since it seems that it is not really checking anything, but simply reporting about the win.  It also does something in addition which is to add a score to the score list.